src/win/util.c

```diff
@@ -1380,6 +1380,12 @@ int uv__getpwuid_r(uv_passwd_t* pwd) {
   }
 
   pwd->shell = NULL;
+  pwd->gecos = uv_malloc(1);
+  if (pwd->gecos == NULL)
+    return UV_ENOMEM;
```

Comment:
When this happens, `homedir` and `username` have both been allocated, so they would need to be released.